src/Api/Helpers.php

```diff
@@ -80,7 +80,7 @@ public static function processRequestException(RequestException $e): \Throwable
             return new ResourceNotFoundException(sprintf($msg, $e->getRequest()->getUri()), 0, $e);
         } elseif ($e->getCode() === 404) {
             // ResourceNotFound, eg. bad fileId, "-1, Microsoft.SharePoint.Client.ResourceNotFoundException"
-            return new BadRequestException(
+            return new ResourceNotFoundException(
```

Comment:
Spadli testy, podla vsetkeho API vracia trocha inu odpoved, ako ked sa to implementovalo, -> zjednotil som to.

https://travis-ci.com/github/keboola/wr-onedrive/builds/204128386

![screenshot-travis-ci com-2020 11 25-{time)](https://user-images.githubusercontent.com/19371734/100211704-bf44c400-2f0c-11eb-81df-899914c74994.png)